modules/component/types/componentProps.ts

```diff
@@ -0,0 +1,23 @@
+import {Action} from '@action-land/core'
+
+/**
+ * Component typing signature
+ * 1. **iState**: Initial state requiered by component to exist.
+ * 2. **oState**: Extends iState may contain additional properties.
+ * 3. **iActions**: Set of actions (messages) handled by component.
```

Comment:
why the `messages` explanation? might be confusing for the target audience... any explanation of actions should be co-located with the `Action` module.